dashboard/config/routes.rb

```diff
@@ -6,6 +6,10 @@ module OPS
 end
 
 Dashboard::Application.routes.draw do
+  resources :featured_projects do
+    delete '/:project_id', action: 'destroy_featured_project'
+  end
```

Comment:
it is confusing to me that the delete route is inside the resource because the `resource` command puts the featured project id into the url, and you're supplying it in the ajax call in the javascript, but then you're not using it in the controller. my recommendation would be to move the `delete` route out of the `resource` wrapper, like this:
```ruby
resource :featured_projects, only: [:create]
delete '/featured_projects/:project_id', action: 'destroy_featured_project'
```
@aoby , does this sound right?

Reply:
That will work, but why create the delete route explicitly if you're already using the `resources` macro?

How about this?
```ruby
resources :featured_projects, only: [:create, :delete]
```

See http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions

Reply:
> why create the delete route explicitly if you're already using the resources macro?

because we're not using the featured_project_id (part of the macro), we're using the project_id (not part of the macro).

Reply:
Gotcha. I missed that. Thanks for the clarification. In that case, I like your suggestion but I think the action name should be more clear. Maybe `destroy_by_project_id`?